deploy-service/teletraanservice/src/main/java/com/pinterest/teletraan/worker/CheckDeployEnvStatusWorker.java

```diff
@@ -0,0 +1,127 @@
+package com.pinterest.teletraan.worker;
+
+import com.pinterest.deployservice.ServiceContext;
+import com.pinterest.deployservice.bean.EnvironBean;
+import com.pinterest.deployservice.dao.DeployDAO;
+import com.pinterest.deployservice.dao.EnvironDAO;
+import com.pinterest.deployservice.dao.UtilDAO;
+import com.pinterest.deployservice.bean.DeployFilterBean;
+import com.pinterest.deployservice.db.DeployQueryFilter;
+import com.pinterest.deployservice.bean.DeployQueryResultBean;
+import com.pinterest.deployservice.bean.*;
+import org.quartz.*;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import java.io.DataOutputStream;
+import java.net.Socket;
+import java.sql.Connection;
+import java.util.*;
+
+public class CheckDeployEnvStatusWorker implements Job {
+    private static final Logger LOG = LoggerFactory.getLogger(CheckDeployEnvStatusWorker.class);
+    private EnvironDAO environDAO;
```

Comment:
It might still be created every time if a new CheckDeployEnvStatusWorker is created each time. That is probably fine. But either way, these variables need to be initialized in the constructor or passed in to the method for best practice programming.